src/Functions/Public/relocation/Invoke-vRAOnboardingPlan.ps1

```diff
@@ -161,7 +161,7 @@
 
                         $ExecuteOnboardingPlan= Invoke-vRARestMethod -Method POST -URI $URI -Body $Body -Verbose:$VerbosePreference
 
-                        CalculateOutput
+                        CalculateOutput($ExecuteOnboardingPlan)
```

Comment:
Should the parameter be positional here and not in ()?

Reply:
resolved in last commit.